src/main/java/hudson/plugins/filesystem_scm/RemoteCopyDir.java

```diff
@@ -5,10 +5,10 @@
 
 import org.apache.commons.io.FileUtils;
 
-import hudson.FilePath.FileCallable;
 import hudson.remoting.VirtualChannel;
+import jenkins.SlaveToMasterFileCallable;
 
-public class RemoteCopyDir implements FileCallable<Boolean> {
+public class RemoteCopyDir extends SlaveToMasterFileCallable<Boolean> {

```

Comment:
Probably also wrong, though the class appears unused anyway so maybe just delete it.